mev_inspect/classifiers/helpers.py

```diff
@@ -84,3 +96,69 @@ def _filter_transfers(
         filtered_transfers.append(transfer)
 
     return filtered_transfers
+
+
+def is_valid_0x_swap(
+    trace: DecodedCallTrace,
+    child_transfers: List[Transfer],
+) -> bool:
+
+    # 1. There should be 2 child transfers, one for each settled leg of the order
+    if len(child_transfers) != 2:
+        raise ValueError(
+            f"A settled order should consist of 2 child transfers, not {len(child_transfers)}."
+        )
+
+    # 2. The function signature must be in the lists of supported signatures
+    if trace.function_signature not in (LIMIT_SIGNATURES + RFQ_SIGNATURES):
+        raise RuntimeError(
+            f"0x orderbook function {trace.function_signature} is not supported"
+        )
+
+    # 3. The swap should not be a child of previous swaps in the block
+
+    return True
+
+
+def _get_taker_token_in_amount(
+    taker_address: str, token_in_address: str, child_transfers: List[Transfer]
+) -> int:
+
+    if taker_address == ANY_TAKER_ADDRESS:
+        for transfer in child_transfers:
+            if transfer.token_address == token_in_address:
+                return transfer.amount
+    else:
+        for transfer in child_transfers:
+            if transfer.to_address == taker_address:
+                return transfer.amount
+    return 0
```

Comment:
is there a case where we ever reach this 0?

Reply:
No, but necessary for mypy to have that return statement